core/src/SYCL/Kokkos_SYCL_Instance.hpp

```diff
@@ -330,8 +330,8 @@ struct sycl::is_device_copyable<
     Kokkos::Experimental::Impl::SYCLFunctionWrapper<Functor, Storage, false>>
     : std::true_type {};
 
-// FIXME_SYCL Remove when this specialization when specializations for
-// sycl::device_copyable also apply to const-qualified types.
+#if (defined(__INTEL_LLVM_COMPILER) && __INTEL_LLVM_COMPILER < 20240000) || \
+    (defined(__LIBSYCL_MAJOR_VERSION) && __LIBSYCL_MAJOR_VERSION >= 7)
```

Comment:
Please confirm that this not meant to be if the SYCL runtime library version is *less than*

Reply:
Ah, good catch. Fixed in 5c1094933c21e832636fc910be00fbae0909fdd0.